float64[] joint_pose
---
bool success          